resources/install/example.sql

```diff
@@ -19,3 +19,14 @@ CREATE TABLE IF NOT EXISTS `activities_activities` (
     KEY `row` (`row`),
     KEY `ip` (`ip`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE IF NOT EXISTS `activities_resources` (
```

Comment:
I was a bit confused about UUID, this is the resource UUID, not the revision Identifier :). Now I understood that the couple revision UUID is the primary key for a revision entry. I've missed this.

Anyway, I've implemented the UUID column as you suggests. This also required some small API changes (see #5). I'm going to try optimizing the table a little bit for improving searches.